### ISSUE TITLE
The problems without the npy file and no errors occured

### ISSUE THREAD

**Issue by Liuhm0710** [Extreme-classification/deepxml#15]
<img width="306" alt="image" src="https://user-images.githubusercontent.com/110908816/206700299-0e4882f7-6919-401d-b3c2-8d708d05deb2.png">
<img width="836" alt="image" src="https://user-images.githubusercontent.com/110908816/206700428-0b01c33e-df67-4b5c-8fd6-66b570716903.png">
Hello, I have run the model provided on Github, but it seens that the preaprations do not support the need.  I checked the web you gave before to another person. And the web is invalid now. With no errors occured, i could not figure out what i miss, maybe for files, for the packages. It will be my pleasure if you could take a look at the problem we provide and help us.

**Comment by kunaldahiya** [comment]
Hi,

Can you please explain the issue in more detail? Are you unable to generate *_X_Xf.txt and *_X_Y.txt files?

**Comment by Liuhm0710** [comment]
Yes we have generated the file according to the need, and the root is Eurlex, one dataset shown as below
<img width="978" alt="image" src="https://user-images.githubusercontent.com/110908816/208299172-9e0d54d9-78ec-4dd3-896a-bb6b7ee1402e.png">
Well the issue is actually no issue but no outcome. We promote the procedure according to the .md, but no issures occured. We consider that 1 it mght be the problem with the npy file, 2 maybe  the structure of our file is wrong, so I put the image as above. again as below.
<img width="220" alt="image" src="https://user-images.githubusercontent.com/110908816/208299504-3631874a-bd73-4956-9a21-9f0ed925ad92.png">
3 the last one may only be the circumstance, but no issue with circumstance occured so, the problems can be concluded as above.

(Thanks for your reading, really thank you)

**Comment by kunaldahiya** [comment]
Could you please name the dataset directory as EURLex-4K (same as dataset name in run command)?

**Comment by Liuhm0710** [comment]
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/110908816/208442505-5dcab14a-df38-4113-ac54-1d44d976b2f6.png">
it still seems no working. 
I tried to extract the run_main.sh from the script directory but still no thing happen. And what is for  the run_datasets.sh for? I could not figure out, does it affect the result or the generation of result?

**Comment by kunaldahiya** [comment]
Let's set up a call sometime to see what's happening?

**Comment by Liuhm0710** [comment]
Well its okay. I'm free this Wed noon and evening and the whole day of Thurs, noon and evenning for Fri and whole day for weekend. I'm a Chinese so maybe you need to convert from my time to get your time. The form of call is a call from the social software or an online meeting may be more convenient? The exact format is determined by your convenience.(THANK YOU)
